oneandone/src/main/java/org/apache/jclouds/oneandone/rest/compute/OneandoneComputeServiceAdapter.java

```diff
@@ -179,9 +179,46 @@
       return new NodeAndInitialCredentials<Server>(updateServer, updateServer.id(), serverCredentials);
    }
 
+   private ServerAppliance findWorkingImage(String imageId, Types.ServerTypeCompatibility serverType) {
```

Comment:
why this name? it seems that it tries to find a `ServerAppliance`, doesn't it?

Reply:
changed